pkg/deploy/devconfig.go

```diff
@@ -174,6 +175,13 @@ func DevConfig(_env env.Core) (*Config, error) {
 			MDMFrontendURL:              ptr.To("https://global.ppe.microsoftmetrics.com/"),
 			MDSDEnvironment:             ptr.To(version.DevGenevaLoggingEnvironment),
 			MsiRpEndpoint:               ptr.To("https://iamaplaceholder.com"),
+			MISEVALIDAUDIENCES: []string{
+				"https://management.core.windows.net/",
```

Comment:
is this consistent every cloud?  Should this be a config?

Reply:
This is for generating devconfig for full RP deployment, so there is no ARM involved, which means MISE will not be needed. I have added a test detail to make it pass validation.